### PR TITLE
Api job url

### LINKED ISSUE #464: API - add items and logs url to listjobs.json
After making a call to listjobs.json, I'd like to download the items or logs related to the finished jobs. Would it make sense to add something like below?

2 new attributes:
* items_url 
* logs_url

```json
{
    "status": "ok",
    "pending": [],
    "running": [],
    "finished": [
        {
            "id": "2f16646cfcaf11e1b0090800272a6d06",
            "project": "myproject", "spider": "spider3",
            "start_time": "2012-09-12 10:14:03.594664",
            "end_time": "2012-09-12 10:24:03.594664",
            "items_url": "/items/myproject/spider3/2f16646cfcaf11e1b0090800272a6d06.jl",
            "log_url": "/logs/myproject/spider3/2f16646cfcaf11e1b0090800272a6d06.log"
        }
    ]
}
```

If so, I could submit a PR for it.

### PR DESCRIPTION
**Pull request by mxdev88**
closes #464

### CODE REVIEW

**Comment by mxdev88** [comment]
@jpmckinney unsure about the linting failure. it was passing on the fork.
https://github.com/mxdev88/scrapyd/actions/runs/3936454424/jobs/6732997024

File: docs/api.rst [hunk 1]
```diff
@@ -212,7 +212,9 @@ Example response::
                 "id": "2f16646cfcaf11e1b0090800272a6d06",
                 "project": "myproject", "spider": "spider3",
                 "start_time": "2012-09-12 10:14:03.594664",
-                "end_time": "2012-09-12 10:24:03.594664"
+                "end_time": "2012-09-12 10:24:03.594664",
+                "logs_url": "/logs/myproject/spider3/2f16646cfcaf11e1b0090800272a6d06.log",
+                "items_url": "/items/myproject/spider3/2f16646cfcaf11e1b0090800272a6d06.jl"
```
Comment (jpmckinney) [inline docs/api.rst:217]: Should `log_url` be singular?

File: scrapyd/tests/conftest.py [hunk 2]
```diff
@@ -47,9 +47,15 @@ def txrequest():
     return Request(http_channel)
 
 
-def common_app_fixture(request):
-    config = Config()
+@pytest.fixture(params=[None, ('scrapyd', 'items_dir', 'items')], ids=["default", "default_with_local_items"])
```
[the remark this answers is not in the document]
Comment (mxdev88) [reply scrapyd/tests/conftest.py:50]: Giving an id to param https://docs.pytest.org/en/stable/reference/reference.html#pytest-fixture

[the remark this answers is not in the document]
Comment (jpmckinney) [reply scrapyd/tests/conftest.py:50]: Ah okay, I usually just let pytest auto-assign.

File: docs/api.rst (hunk 1 above)
Comment (mxdev88) [reply docs/api.rst:217]: Yes you're probably right.

Comment (mxdev88) [reply docs/api.rst]: I've renamed `logs_url` into `log_url`.

**Review by jpmckinney — APPROVED** [review]

**Comment by jpmckinney** [comment]
Thank you!

**Pull request merged**